### ISSUE TITLE
Character encoding

### ISSUE THREAD

**Issue by scientiacloud** [kellyselden/ember-i18n-csv#24]
When exporting a language file to CSV the special characters are not copied across properly:
    `export default {
        weekdays: ['Terça',  'Sábado'],
    }`
becomes:
    `weekdays: ['TerÃ§a', 'SÃ¡bado']`

Am I doing something wrong?

**Comment by kellyselden** [comment]
> Am I doing something wrong?

Probably not. I will add those to my test cases and see if I can figure out what's going on.

**Comment by kellyselden** [comment]
I added the keys to my test cases here #26 and my tests still passed. At this point I'm not sure what the problem is. Did you open alter the csv using Excel or OpenOffice or change the encoding?

**Comment by FredUK** [comment]
Hi there, I have not changed anything. After I ran `ember-i18n-csv to-csv --locales-path=publish/ --csv-path=i18n.csv` and opened the CSV file in Excel that's how it showed up. 

However I just created an empty excel document and went to "Data - Import - From text", selectd the CSV file and go to the "Text Import wizard" screen and in there the "file origin" was set to Windows (ANSI). If I change it to UTF-8 then everything is fine. Not sure why Excel is defaulting it to Windows (ANSI) but apparently I found out what the problem was. Thanks for your speedy reply.

**Comment by FredUK** [comment]
Extra update: After importing the CSV data as UTF8 the characters now look fine. Saving the file and re-opening it again still shows properly. 

However when trying to convert back to js with "to-js" command the javascript will strip out the special characters and replace them with squares. Check screenshot below:
![image](https://cloud.githubusercontent.com/assets/11888256/12390712/69481424-bdda-11e5-950c-4cdbedaf5672.png)

Although the tests pass I assume it can only be the way the CSV file is being created and the data inserted into it? Would you be able to apart from your tests passing also generate the CSV file and check it manually to see if the characters are still properly displayed?

**Comment by kellyselden** [comment]
I think your problem is Excel related. If you generate the CSV, then reimport the CSV again without opening it in Excel, it works right?

**Comment by FredUK** [comment]
Hi Kelly, you're correct. I have just tried to open the unchanged file in other editors (how didn't I think of that before??!) and they all display it properly. Also converting it back to json works flawlessly.

As soon as it opened in Excel it show's gibberish with the special characters. If I tell excel to import the CSV as UTF-8 it then shows properly in Excel but saving it will ruin it for all other programs.

Basically it can't read the translation file properly while all other text editors can; you tell it to force UTF-8 and it works fine. You tell it to save it as a new CSV file (still works fine on excel) but not for any other program..Pff..

My apologies for wasting your time. Thanks a lot.

**Comment by kellyselden** [comment]
No problem, I would welcome an addition to our readme document the troubles with Excel for other users if you have time.

**Comment by kellyselden** [comment]
@scientiacloud @FredUK I have added Excel instructions that have worked for me https://github.com/kellyselden/ember-i18n-csv/commit/0ab878710b955d87bbbfb3d7fd4911abb95a6776

**Comment by FredUK** [comment]
Thanks, in the end I decided to use Google Docs (Sheets) to maintain the translation files instead of using Office :+1: . This way is also easier to add collaborators to the translations.

**Comment by kellyselden** [comment]
@FredUK Sounds good.
